frontend/src/components/auth/UpdateProfileForm.vue

```diff
@@ -0,0 +1,213 @@
+                        color="#3C57DE"
+                    >
+                        Save
+                    </VBtn>
+                </VCardActions>
+            </VContainer>
+        </VFlex>
+    </VContent>
+</template>
+
+<script>
+    import {mapGetters, mapActions} from 'vuex';
+    import {UPDATE} from '@/store/modules/auth/types/actions';
+    import {GET_AUTHENTICATED_USER} from '@/store/modules/auth/types/getters';
+
+    export default {
+        data() {
+            return {
+                editUser: {
+                    name: '',
+                    email: '',
+                    password: '',
+                },
+                confirmPassword: '',
+                valid: false,
+                showPassword: false,
+                showConfirmPassword: false,
+                nameRules: [
+                    v => !!v || 'Field full name is required',
+                    v => (v && v.length >= 3) || 'Enter the correct information'
+                ],
+                emailRules: [
+                    v => !!v || 'E-mail is required',
+                    v => /.+@.+\..+/.test(v) || 'E-mail must be valid',
+                ],
+                passwordRules: [
+                    v => !!v || 'Password is required',
+                    v => (v && v.length >= 8) || 'Password must be equal or more than 6 characters'
+                ],
+                confirmPasswordRules: [
+                    v => !!v || 'Password is required',
+                    v => v === this.editUser.password || 'Password should match'
+                ]
+            };
+        },
+
+        created() {
+            this.editUser = {
+                ...this.getUser
+            };
+        },
+
+        computed: {
+            ...mapGetters('auth', {
+                getUser: GET_AUTHENTICATED_USER
+            }),
+            passwordVisibility: function() {
+                return this.showPassword ? 'visibility' : 'visibility_off';
+            },
+            confirmPasswordVisibility: function () {
+                return this.showConfirmPassword ? 'visibility' : 'visibility_off';
+            },
+            passwordType: function () {
+                return this.showPassword ? 'text' : 'password';
+            },
+            confirmPasswordType: function () {
+                return this.showConfirmPassword ? 'text' : 'password';
+            }
+        },
+
+        methods: {
+            ...mapActions('auth', {
+                update: UPDATE
+            }),
+
+            onSave() {
+                if (this.$refs.form.validate()) {
+                    this.update(this.editUser)
+                        .catch((error) => {
+                            alert(error.message);
+                        })
+                }
+            },
+        }
+    };
+</script>
+
+<style lang="scss" scoped>
+.edit-form-input {
+    background: #FFFFFF;
+    border: 1px solid rgba(18, 39, 55, 0.11);
```

Comment:
lets remove this border styles for now, we may need to configure input styling  globally later